genealogy/geneweb/Portfile

```diff
@@ -1,29 +1,48 @@
 PortSystem 1.0
+PortGroup           github 1.0
 
 name			geneweb
-version			5.02
-revision		3
+version			7.1-beta
+revision        0
+github.setup    ${name} ${name} ${version} v
 categories		genealogy
 maintainers		nomaintainer
-description		Sophisticated Genealogy System 
+description		Sophisticated Genealogy System
 long_description	GeneWeb is sophisticated genealogy software with a \
 			web interface.  It provides efficient relationship and consanguinity \
 			computing, support for very large database (i.e., hundreds of thousands \
 			of entries), and more.
 
-homepage		http://cristal.inria.fr/~ddr/GeneWeb
-master_sites		http://cristal.inria.fr/~ddr/GeneWeb/ftp/Src \
-			ftp://ftp.inria.fr/INRIA/Projects/cristal/${name}/Src
-distfiles		${name}-${version}.tgz
-checksums		md5 e8c6c789a9dd2f9155bf6eefa8786d01 \
-			sha1 e040496b3ce106cc7a508360cd0e4268bcb3e7b1 \
-			rmd160 fd284630e11957cf1265bfce9207e46d449a83ec
-platforms		darwin
-
-depends_build		port:ocaml \
-			port:camlp5
-
-patchfiles		patch-5.02-1
+homepage		https://geneweb.tuxfamily.org/wiki/GeneWeb
+extract.suffix      .zip
+checksums           rmd160  3745f4476378fb90cd21f628ae6a2e93dd0dae21 \
+                    sha256  527bc5a59e378cad144449800d214c641d775f5269716f3c40e68c5de87eab9a \
+                    size    14465775
+
+license             GPL
```

Comment:
Further alignment issues with `license`, `extract.suffix`, and `checksums`

Reply:
Fixed in a second commit as suggested.